api/api-rules/violation_exceptions.list

```diff
@@ -251,6 +251,7 @@ API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,V
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,VolumeConfiguration,PersistentVolumeRecyclerConfiguration
 API rule violation: names_match,k8s.io/kube-proxy/config/v1alpha1,KubeProxyConfiguration,IPTables
 API rule violation: names_match,k8s.io/kube-proxy/config/v1alpha1,KubeProxyConfiguration,NFTables
+API rule violation: names_match,k8s.io/kubelet/config/v1beta1,KubeletConfiguration,CrashLoopBackOff
```

Comment:
This is an immediate smell  --  the name mismatch is comment on in the API file.